src/xenialdan/Slenderman/entities/Slenderman.php

```diff
@@ -4,93 +4,101 @@
 use pocketmine\plugin\Plugin;
-use pocketmine\scheduler\PluginTask;
-use pocketmine\Server;
+use pocketmine\scheduler\Task;
 use xenialdan\skinapi\API;
 use xenialdan\Slenderman\Loader;
 
-class Slenderman extends Human{
+class Slenderman extends Human
+{
 
-	public function __construct(Level $level, CompoundTag $nbt){
-		$path = Loader::getInstance()->getSkin();
-		$img = imagecreatefrompng($path . '.png');
-		$skin = new Skin('Slenderman', API::fromImage($img), "", "geometry.humanoid.custom", "");
-		imagedestroy($img);
-		$this->setSkin($skin);
-		parent::__construct($level, $nbt);
-		$this->setDataProperty(self::DATA_SCALE, self::DATA_TYPE_FLOAT, 1.3);
-		$this->setImmobile();
-		$this->setMaxHealth(200);
-		$this->setHealth(200);
-	}
+    public function __construct(Level $level, CompoundTag $nbt)
+    {
+        $path = Loader::getInstance()->getSkin();
+        $img = imagecreatefrompng($path . '.png');
+        $skin = new Skin('Slenderman', API::fromImage($img), "", "geometry.humanoid.custom", "");
+        imagedestroy($img);
+        $this->setSkin($skin);
+        parent::__construct($level, $nbt);
+        $this->getDataPropertyManager()->setFloat(self::DATA_SCALE, 1.3);
+        $this->setImmobile();
+        $this->setMaxHealth(200);
+        $this->setHealth(200);
+    }
 
-	public function triggerTeleport(Player $player){
-		$level = $player->getLevel();
-		$allowedground = [Block::PODZOL, Block::STAINED_CLAY, Block::DIRT, Block::CONCRETE_POWDER];
-		$min = 10;
-		$max = 20;
-		//teleport it to a save spot min to max blocks away
-		$tries = 0;
-		do{
-			$newpos = $level->getSafeSpawn($this->add((mt_rand(0, 1) === 0 ? mt_rand(-$max, -$min) : mt_rand($min, $max)), (mt_rand(0, 1) === 0 ? mt_rand(-$max, -$min) : mt_rand($min, $max)), (mt_rand(0, 1) === 0 ? mt_rand(-$max, -$min) : mt_rand($min, $max))));
-			$tries++;
-		} while ($tries <= 10 && !in_array($level->getBlock($newpos)->getSide(Vector3::SIDE_DOWN)->getId(), $allowedground));
-		$this->teleport($newpos);
-		$this->sendSkin($level->getPlayers());//Because the client messes this up
-		$this->lookAt($player);
+    public function triggerTeleport(Player $player)
+    {
+        $level = $player->getLevel();
+        $allowedground = [Block::PODZOL, Block::STAINED_CLAY, Block::DIRT, Block::CONCRETE_POWDER];
+        $min = 10;
+        $max = 20;
+        //teleport it to a save spot min to max blocks away
+        $tries = 0;
+        do {
+            $newpos = $level->getSafeSpawn($this->add((mt_rand(0, 1) === 0 ? mt_rand(-$max, -$min) : mt_rand($min, $max)), (mt_rand(0, 1) === 0 ? mt_rand(-$max, -$min) : mt_rand($min, $max)), (mt_rand(0, 1) === 0 ? mt_rand(-$max, -$min) : mt_rand($min, $max))));
+            $tries++;
+        } while ($tries <= 10 && !in_array($level->getBlock($newpos)->getSide(Facing::DOWN)->getId(), $allowedground));
+        $this->teleport($newpos);
+        $this->sendSkin($level->getPlayers());//Because the client messes this up
```

Comment:
Sending the skin causes client crashes under specific circumstances. (#blameMojang) Since this workaround isn't necessary anymore, I suggest removing it.